contrib/lib/src/lib.rs

```diff
@@ -40,7 +40,7 @@
 //! This crate is expected to grow with time, bringing in outside crates to be
 //! officially supported by Rocket.
 
-#[allow(unused_imports)] #[macro_use] extern crate log;
+#[allow(unused_imports)] #[macro_use] extern crate tracing;
```

Comment:
What do you suggest changing here? I assume `#[macro_use]` was used to avoid needing to add `use` statements to each file, and I'm not aware of another way to do this (aside from making all of the different logging macros top-level exports from `rocket`, and `#[macro_use] extern crate rocket;`)